app/assets/stylesheets/application.scss

```
@@ -49,6 +49,20 @@
 .card-image{
   height: 390px;
 }
+
+
+.about-cards{
+  width: 40%;
+  text-align: center;
+  vertical-align: center;
+  background-color: whitesmoke;
+  margin: 1% 40% 10% 30%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  
+  }
```

```suggestion
}
```
indentation of curly braces should align with where the selector is provided